addons/gut/gui/gut_config_gui.gd

```diff
@@ -272,8 +272,8 @@ func get_config_issues():
 	if(!has_directory):
 		to_return.append('You do not have any directories set.')
 
-	if(_cfg_ctrls['prefix'].text == ''):
-		to_return.append("You must set a Script prefix or GUT won't find any scripts")
```

Comment:
As I said in the top description, I don't think this warning is actually true.

You can set it to `''` in `test_gut_directory.gd` and the tests still pass.